moveit_planners/stomp/industrial_collision_detection/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(industrial_collision_detection)
+
+add_definitions("-std=c++11")
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  cmake_modules
+  pluginlib
+  moveit_core
+)
+
+find_package(Boost REQUIRED)
+find_package(Eigen REQUIRED)
+find_package(console_bridge REQUIRED)
+find_package(octomap REQUIRED)
+find_package(urdfdom REQUIRED)
+find_package(urdfdom_headers REQUIRED)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(LIBFCL REQUIRED fcl)
+find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
+set(LIBFCL_LIBRARIES "${LIBFCL_LIBRARIES_FULL}")
+
+###################################
+## catkin specific configuration ##
+###################################
+## The catkin_package macro generates cmake config files for your package
+## Declare things to be passed to dependent projects
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
```

Comment:
side note: i think you mistakenly added this comment to the wrong location

it sounds like the changes in the industrial_collision_detection package should just be merged into the original FCL collision detection codebase within moveit - otherwise we have a lot of duplicate code that does almost the same thing